docs/src/content/docs/ja/guides/components.mdx

```diff
@@ -116,6 +116,50 @@ import { Card, CardGrid } from '@astrojs/starlight/components';
 
 :::
 
+### リンクカード
+
+`<LinkCard>`を使用すると、他のページへのリンクを目立たせることができます。
+
+`<LinkCard>`は`title`と[`href`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#href)属性を必須とします。短い`description`や、`target`など他のリンク属性も指定できます。
```

Comment:
I think we could link to the same MDN docs but in the Japanese translation here:

```suggestion
`<LinkCard>`は`title`と[`href`](https://developer.mozilla.org/ja/docs/Web/HTML/Element/a#href)属性を必須とします。短い`description`や、`target`など他のリンク属性も指定できます。
```